appveyor.yml

```diff
@@ -129,4 +129,24 @@ on_success:
   - set ARTIFACT=%GROONGA_INSTALL_FOLDER%.zip
   - 7z a %ARTIFACT% %FULL_GROONGA_INSTALL_FOLDER%
   - ps: Push-AppveyorArtifact $Env:ARTIFACT
+  - if "%ARCH%" == "amd64" set ARCH=x64
+  - ps: |
+      if ($Env:VS_VERSION -eq "15") {
+        $Env:REDIST_VERSION = (Get-Content "C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Auxiliary\Build\Microsoft.VCRedistVersion.default.txt")
+        $Env:VCREDIST_VCRUNTIME = "C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Redist\MSVC\$Env:REDIST_VERSION\$Env:ARCH\Microsoft.VC141.CRT\vcruntime140.dll"
+        $Env:VCREDIST_MSVCP = "C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Redist\MSVC\$Env:REDIST_VERSION\$Env:ARCH\Microsoft.VC141.CRT\msvcp140.dll"
+      }
+      if ($Env:VS_VERSION -eq "14") {
+        $Env:VCREDIST_VCRUNTIME = "C:\Program Files (x86)\Microsoft Visual Studio $Env:VS_VERSION.0\VC\redist\$Env:ARCH\Microsoft.VC140.CRT\vcruntime140.dll"
```

Comment:
If `"C:\Program Files (x86)\Microsoft Visual Studio $Env:VS_VERSION.0\VC\redist\$Env:ARCH\"` is path of common for VisualStudio2014 and 2012, we should set it in a variable.
Because we easily to read these codes.

Reply:
It is reasonable to define variable.